tempesta_fw/t/unit/test_frang.c

```diff
@@ -0,0 +1,399 @@
+/**
+ *  		Tempesta FW
+ *
+ * Copyright (C) 2015 Tempesta Technologies, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * his program; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ */
+
+#include <linux/inet.h>
+
+#include "../../classifier.h"
```

Comment:
Are you sure? https://github.com/natsys/tempesta/blob/63295ae28c74852836cdc03f390c42eb5ab55a69/tempesta_fw/gfsm.h#L121
but gfsm.h include indirectly - you can not add it